cli/azd/pkg/convert/util.go

```diff
@@ -0,0 +1,11 @@
+package convert
```

Comment:
What about moving `RefOf` from the `mocks` package to here at some point?  In @vhvb1989  PRs for the `azdo` stuff I think we could have simplified some code by having something like `RefOf` even outside the context of mocking.